lib/govuk_app_config/railtie.rb

```diff
@@ -3,5 +3,9 @@ class Railtie < Rails::Railtie
     config.before_initialize do
       GovukLogging.configure if Rails.env.production?
     end
+
+    if Rails.env.development? && config.respond_to?(:hosts)
+      config.hosts << "#{File.basename(Rails.root)}.dev.gov.uk"
```

Comment:
Maybe try and use `Env.fetch('GOVUK_APP_NAME')`, and maybe fall back to `File.basename(Rails.root)`?

That's a bit more flexible, and can avoid depending too much on where the `Rails.root` is.

Reply:
Code-wise, line 8, this will work. It's just not _getting in_ to the app.